app-emulation/retroarch/retroarch-1.7.1.ebuild

```diff
@@ -5,28 +5,29 @@ EAPI=6
 
 inherit flag-o-matic
 
-DESCRIPTION="A sophisticated libretro frontend for emulators, game engines and media players."
-HOMEPAGE="http://retroarch.com/"
+DESCRIPTION="A sophisticated libretro frontend for emulators, game engines and media players"
+HOMEPAGE="https://retroarch.com/"
 SRC_URI="https://github.com/libretro/RetroArch/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
+IUSE="-alsa cg +opengl +pulseaudio -sdl X"
+KEYWORDS="~amd64 ~arm ~x86"
 LICENSE="GPL-3"
 SLOT="0"
-KEYWORDS="~amd64 ~x86 ~arm"
-IUSE="+opengl +pulseaudio -alsa -sdl -X cg"
+
 
 DEPEND="
 	dev-libs/libxml2
 	>=media-libs/freetype-2.8
+	alsa? ( media-sound/alsa-utils )
 	opengl? ( virtual/opengl )
-	X? ( x11-libs/libX11 x11-apps/xinput )
 	pulseaudio? ( media-sound/pulseaudio )
-	alsa? ( media-sound/alsa-utils )
-	sdl? ( media-libs/libsdl2 )"
+	sdl? ( media-libs/libsdl2 )
+	X? ( x11-libs/libX11 x11-apps/xinput )"
 
 RDEPEND="
 	${DEPEND}
-	x86? ( cg? ( media-gfx/nvidia-cg-toolkit ) )
-	amd64? ( cg? ( media-gfx/nvidia-cg-toolkit ) )"
+	amd64? ( cg? ( media-gfx/nvidia-cg-toolkit ) )
+	x86? ( cg? ( media-gfx/nvidia-cg-toolkit ) )"
```

Comment:
no, you do it wrong, arch flags should not be used unless special cases, which is not the case, make it only depends on cg, if it does not work with arm, well, you have to p.use.mask it on arm

Reply:
Removed arch flags.
Added the USE flag as masked in profiles/arch/arm/package.use.mask and profiles/arch/arm64/package.use.mask.